docs/guides/processing-page-builder-content.md

```diff
@@ -0,0 +1,365 @@
+# How to process page-builder content in Frontity
+
+Video: https://www.youtube.com/watch?v=qOfENWKR7EE&list=PLC9teX20GdrTBeOzSwE-bFW-MbBEUwowS&index=6&t=159s
```

Comment:
What is the purpose of this link?

Reply:
Oh, that's just there for my reference - it will be removed before we publish it.